tests/oscillating_collimator_let.py

```diff
@@ -83,3 +85,22 @@ def test_WHEN_input_values_cause_discriminant_to_be_negative_THEN_discriminant_p
 
         # Assert
         self.ca.assert_that_pv_is_number(DISCRIMINANT, 1.0)
+
+    def test_WHEN_laser_unstable_THEN_collimator_is_reported_moving(self):
+        # Act
+        # in normal operations the radius is not dynamic so set it first so it is considered in future calcs
+        # Set 0 first then 1 so buffer contents are definitively inconsistent
+        self.ca_mot.set_pv_value("DMC01:Galil0Bi5_STATUS", 0)
+        self.ca_mot.set_pv_value("DMC01:Galil0Bi5_STATUS", 1)
```

Comment:
Since your buffer scans once a second, if it was already full of 1s you may miss the intermediate `0` state?

Best to wait a short time (e.g. 1s) between these two sets, or actually monitor the buffer and make sure that it has different values in it (you may need to use `assert_pv_value_causes_func_to_return_true`, or a similar custom assertion, for this).